### PR TITLE
Adds options to filter labels and bumps node version to 16

### PR DESCRIPTION
**Pull request by DarshitChanpura**
Adds an option to include or exclude labels when copying

### CODE REVIEW

**Comment by DarshitChanpura** [comment]
@michalvankodev Feel free to review and merge this.

File: action.yml [hunk 1]
```diff
@@ -11,6 +11,12 @@ inputs:
   from-title:
     description: 'To parse the issue numbers from title, a boolean'
     required: false
+  labels-to-copy:
```
Comment (michalvankodev) [inline action.yml:14]:
```suggestion
  include-labels:
```
It's important to get this API right. But however I try, I fail to come up with something that will also be descriptive, simple and categorize the parameter to `labels`.

If I didn't have the context, it'd be sufficient for me to just have `include` as the name for this input.

What do you think? Can you come up with an alternative or a better argument for `labels-to-copy`?

File: src/index.ts [hunk 2]
```diff
@@ -69,8 +71,21 @@ async function run() {
 
   const labels = uniq(
     connectedLabelsResponses.reduce<string[]>((acc, response) => {
-      const issueLabels = response.data.map((label) => label.name)
-      return [...acc, ...issueLabels]
+      const issueLabels = response.data.map((label) => label.name);
+      // Filter out unwanted labels and keep only the ones that are needed
+      const filteredLabels = issueLabels.filter(label => {
+          if (labelsToCopy.length > 0 && !labelsToCopy.includes(label)) {
+              // Label not in labelsToCopy
```
Comment (michalvankodev) [inline src/index.ts:78]:
```suggestion
              // Label not in `labelsToCopy`
```

You can use markdown formatting in code comments. Formatting will also be applied in your text editor.

File: src/index.ts [hunk 3]
```diff
@@ -69,8 +71,21 @@ async function run() {
...
+              return false;
+          }
+          if (labelsToExclude.length > 0 && labelsToExclude.includes(label)) {
+              // Label in labelsToExclude
```
Comment (michalvankodev) [inline src/index.ts:82]:
```suggestion
              // Label in `labelsToExclude`
```

File: README.md
Comment (michalvankodev) [inline README.md:1]: It would be useful to also update the rest of the examples with the `v1.3.0` version.

File: action.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 name: 'copy-issue-labels'
 description: 'Copy labels from linkend issues'
-author: 'Michal Vanko'
+author: 'Michal Vanko, Darshit Chanpura'
```
Comment (michalvankodev) [inline action.yml:3]:
```suggestion
author: 'Michal Vanko'
```
I'm very thankful for every PR and suggestion. I hope this won't sound ungrateful, but I won't consider you an author. It wouldn't be fair to me, who came up with the idea and the need for this action, and also to other contributors who also contributed to the repository. You will still be displayed in this repository as a contributor.

**Review by michalvankodev — CHANGES_REQUESTED** [review]
Thank you very much for this useful feature. I appreciate the PR.

I've pointed out some useful small changes and also I'd like to have a "just right" API for this feature. We need to think if there aren't better names for these inputs.

Let me know what you think.

File: action.yml (hunk 4 above)
Comment (DarshitChanpura) [reply action.yml:3]: np

File: README.md
Comment (DarshitChanpura) [reply README.md:1]: okay

File: action.yml (hunk 1 above)
Comment (DarshitChanpura) [reply action.yml:14]: `labels-to-include` ?

Comment (michalvankodev) [reply action.yml]: That should be fine! Thanks, I will release this with the other PR together!

**Review by michalvankodev — APPROVED** [review]

**Pull request merged**